apps/condo/domains/notification/constants/constants.js

```diff
@@ -79,59 +94,99 @@ const MESSAGE_META = {
         type: { defaultValue: 'UNKNOWN', required: true },
         data: { defaultValue: null, required: true },
     },
-    [MESSAGE_FORWARDED_TO_SUPPORT]: {
+    [MESSAGE_FORWARDED_TO_SUPPORT_TYPE]: {
         dv: { defaultValue: '', required: true },
         text: { defaultValue: null, required: true },
         os: { defaultValue: null, required: true },
         appVersion: { defaultValue: null, required: true },
         organizationsData: { defaultValue: [], isRequired: false },
         attachments: { defaultValue: [], isRequired: false },
     },
+    [TICKET_ASSIGNEE_CONNECTED_TYPE]: {
+        dv: { defaultValue: '', required: true },
+        ticketId: { defaultValue: '', required: true },
+        ticketNumber: { defaultValue: '', required: true },
+        userId: { defaultValue: null, required: true },
+    },
+    [TICKET_EXECUTOR_CONNECTED_TYPE]: {
+        dv: { defaultValue: '', required: true },
+        ticketId: { defaultValue: '', required: true },
+        ticketNumber: { defaultValue: '', required: true },
+        userId: { defaultValue: null, required: true },
+    },
+    [TICKET_STATUS_CHANGED_TYPE]: {
+        dv: { defaultValue: '', required: true },
+        ticketId: { defaultValue: '', required: true },
+        ticketNumber: { defaultValue: '', required: true },
+        ticketStatus: { defaultValue: '', required: true },
+        userId: { defaultValue: null, required: true },
+    },
 }
 
 const MESSAGE_SENDING_STATUS = 'sending'
 const MESSAGE_RESENDING_STATUS = 'resending'
 const MESSAGE_PROCESSING_STATUS = 'processing'
 const MESSAGE_ERROR_STATUS = 'error'
 const MESSAGE_DELIVERED_STATUS = 'delivered'
+const MESSAGE_READ_STATUS = 'read'
```

Comment:
It's not a part of the delivery transport status. Move it to another field

Reply:
removed